src/app/home/index.njk

```diff
@@ -116,13 +116,13 @@
                 <td class="govuk-table__cell">{{ gar.departureTime }} </td>
                 <td class="govuk-table__cell">{{ gar.departurePort | upper }} </td>
                 <td class="govuk-table__cell">{{ gar.arrivalPort | upper }} </td>
-                <td id="garid-{{ name }}-{{ loop.index }}" class="govuk-table__cell">{{ gar.garId }} </td>
+                <td id="garid-{{ name }}-{{ loop.index }}" class="govuk-table__cell">
```

Comment:
Screen reader friendly output of the GAR ID. This is a macro as I sense this might be needed for other screens.